test/python/result/test_quasi.py

```diff
@@ -84,6 +84,18 @@ def test_bin_no_prefix_quasi_bin_out(self):
         quasi = QuasiDistribution(qprobs)
         self.assertEqual(qprobs, quasi.binary_probabilities())
 
+    def test_bin_no_prefix_w_heading_zero_quasi_bin_out(self):
```

Comment:
Could you add a test where the probabilities are e.g. {0: 1}, so the inferred num_bits is 1, and then we ask for `binary_probabilities(num_bits=5)` (or some other number >1)?

Reply:
done. `test_bin_no_prefix_out_padded`